locationdelegation/src/main/java/com/google/androidbrowserhelper/locationdelegation/TrustedWebActivityLocationCallback.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.androidbrowserhelper.locationdelegation;
+
+import android.os.Bundle;
+
+/**
+ * A wrapper for {@link androidx.browser.trusted.TrustedWebActivityCallbackRemote} that we can
+ * construct and use for testing.
+ */
+public interface TrustedWebActivityLocationCallback {
```

Comment:
I'm kinda wondering whether we want a more general `TrustedWebActivityCallbackRemoteWrapper` or something which can then be used from all ABH libraries. This seems fine for now though.

Reply:
That is a good point.